### DISCUSSION TITLE
Nextcloud-AIO-Guide

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#962] [Ideas] [answered]
A guide like [this ](https://szaimen.github.io/Nextcloud-NAS-Guide/)for Nextcloud AIO would be perfect for anyone hosting at home. @szaimen what do you think about it?

**Comment by szaimen** [comment, marked as answer]
I fear this is out of scope as I am very busy with other tasks and the NAS-guide is btw only a hobby project of mine...
